### ISSUE TITLE
jsdoc-informative-docs generates a report whenever you use @class

### ISSUE THREAD

**Issue by ThosRTanner** [gajus/eslint-plugin-jsdoc#1039]
## Expected behavior
```js
/** Some Description
 *
 * @class 
 *
 * @param {number} value - Some useful text
 */
function MyAmazingThing(value)
{
    this.thing = value;
}
```

should not produce a "This description only repeats the name it provides" warning on the `@class` line. The jsdoc documentation on `@class` doesn't require anything else on the line, and to be honest, the only description that is applicable to some classes is the short one.

## Actual behavior

The `@class` line generates the above warning.

## ESLint Config

```
---
plugins:
- jsdoc

rules:

  "jsdoc/informative-docs": warn
```

## ESLint sample

See above

## Environment

- Node version: v18.16.0

- ESLint version v8.38.0

- `eslint-plugin-jsdoc` version: 43.0.7

**Comment by brettz9** [comment]
Since the issue here apparently relates to the algorithm for determining what is "informative", please file your concern with https://github.com/JoshuaKGoldberg/are-docs-informative/issues . I figure the exact decision about what should be allowed may end up being subjective, so my recommendation is to consider whether your recommendation can be phrased as a choice of how to make the behavior more configurable (beyond the configuration that is already offered by the package and our use of it).

If the package ends up allowing more configuration to accommodate differences of opinion about what is informative or not, feel free to submit an issue here so we can pass on that configuration.

Closing as I think that is the approach to take, but feel free to comment further as needed.

**Comment by JoshuaKGoldberg** [comment]
Hmm, I think this is an issue in `eslint-plugin-jsdoc`. Right now, `informative-docs` isn't trimming the text it passes to `are-docs-informative`. The `Boolean(text)` check that's supposed to filter out blank text isn't accounting for whitespace. That means when there's a tag like `@class`, it calls the equivalent of:

```js
areDocsInformative('\n', 'MyAmazingThing')
```

...which is false.

**Comment by ThosRTanner** [comment]
the fix works. thank you
